src/script/components/utils/InViewport.tsx

```diff
@@ -58,15 +60,23 @@ const InViewport: React.FC<InViewportParams & React.HTMLProps<HTMLDivElement>> =
     const triggerCallbackIfVisible = () => {
       if (inViewport && visible) {
         onVisible();
-        releaseTrackers();
+
+        if (!onVisibilityLost) {
+          releaseTrackers();
+        }
```

Comment:
We don't want to release the trackers after detecting element's visibility (we want to detect its visibility lost later).